src/UI/templates/default/Button/tpl.primary.html

```diff
@@ -0,0 +1 @@
+<button type="button" class="btn btn-default btn-primary" href="{ACTION}" {DISABLED}>{LABEL}</button>
```

Comment:
As I wrote, the designer already gets separate css classes that should be named in the kitchen sink meta data. I see that this is a trade-off between designer needs and maintainability of our UI components. I would vote for not duplicating these things.